python/ray/state.py

```diff
@@ -423,7 +424,10 @@ def get_strategy(strategy):
                 placement_group_info.placement_group_id),
             "name": placement_group_info.name,
             "bundles": {
-                bundle.bundle_id.bundle_index: bundle.unit_resources
+                # The value here is needs to be dictionarified
```

Comment:
This is because this bundle should be stored in `class PlacementGroup`'s attribute, and this class should be able to be passed around tasks/actors.